.gitignore

```diff
@@ -3,3 +3,4 @@
 /www
 /www.new
 /www.old
+.DS_Store
```

Comment:
Please don't add this, as otherwise you'd find yourself the need to add it to every single git checkout you have on your system. Instead use https://git-scm.com/docs/git-config#git-config-coreexcludesFile
```
mkdir -p $HOME/.config/git
echo ".DS_Store" >> $HOME/.config/git/ignore
```